src/components/PetAttributeRows.jsx

```diff
@@ -10,21 +10,39 @@ import PetAttributes from "./PetAttributes";
 import PropTypes from "prop-types";
 import React from "react";
 
+const options = {
+  year: "numeric",
+  month: "numeric",
+  day: "numeric",
+};
+
 const PetAttributeRows = (props) => {
-  const { attributes, animalId, gender } = props;
+  const { attributes, animalId, gender, petType, dateAdoptedRedeemed } = props;
+  const shelterArrivalDate = new Date(dateAdoptedRedeemed).toLocaleDateString(
+    "en-US",
+    options
+  );
 
   return (
     <Table className="table-fixed">
       <TableBody>
         <TableRow key={animalId}>
-          <TableHeadCell>Animal Id</TableHeadCell>
+          <TableHeadCell>Animal ID</TableHeadCell>
```

Comment:
Was this intentional?

Reply:
Yes Jason said that in prod the ID is all capital and to change it to match